addons/pharos-cloud-controller/resources/03-cluster-role-pvl-controller.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Do we really need this?

Reply:
Yes. Unfortunately PV stuff is baked into the cloud-controller lower layers:

If I remove those:
```
ERROR: logging before flag.Parse: E0322 10:56:49.597378       1 reflector.go:205] github.com/kontena/pharos-cloud-controller/vendor/k8s.io/kubernetes/pkg/controller/cloud/pvlcontroller.go:110: Failed to list *v1.PersistentVolume: persistentvolumes is forbidden: User "system:serviceaccount:kube-system:pvl-controller" cannot list resource "persistentvolumes" in API group "" at the cluster scope

```